cohort_4/playbook/ACTIVITIES.md

```diff
@@ -2,6 +2,9 @@
 **Table of Contents**
 
 - [Hacker arrival](#hacker-arrival)
+- [Energizers](#energizers)
+  - [Where The West Wind Blows](#where-the-west-wind-blows)
+  - [I Love My Neighbor Who ___](#i-love-my-neighbor-who-___)
```

Comment:
Can you indent this 4 spaces like the rest of the TOC? I know it doesn't follow our markdown style guide, but making this 4 spaces would be consistent with what we currently have.